planetiler-custommap/README.md

```diff
@@ -0,0 +1,86 @@
+# Configurable Planetiler Schema
+
+It is possible to customize planetiler's output from configuration files.  This is done using the parameter:
+`--schema=schema_file.yml`
+
+The schema file provides information to planetiler about how to construct the tiles and which layers, features, and attributes will be posted to the file.  Schema files are in [YAML](https://yaml.org) format.
+
+For examples, see [samples](src/main/resources/samples) or [test cases](src/test/resources/validSchema).
```

Comment:
Even though planetiler is 0.4, I think we should also probably put a note in here that the format may change as we iterate on it, to set expectations around API stability.